### ISSUE TITLE
Circle Template Animations not working on 0.8.6 - console error

### ISSUE THREAD

**Issue by benbarbour** [otigon/automated-jb2a-animations#82]
I'm using foundry 0.8.6. I don't know if it worked before, never tried on an earlier version. Every time I cast a spell with an animation set to "Explosives (Circle Template)" I get this console error:

```
Uncaught TypeError: data.position is undefined
    oncanplay AutoAnimationsLayer.js:83
    playVideo AutoAnimationsLayer.js:74
    playVideo AutoAnimationsLayer.js:42
    SpellAnimation explosion-template.js:85
    setTimeout handler*SpellAnimation explosion-template.js:84
    cast explosion-template.js:91
    explodeTemplate explosion-template.js:125
    revItUp5eCore autoAnimations.js:595
    _call lib-wrapper.js line 1 > eval:4
    callAll foundry.js:153
    callback foundry.js:8667
    documents foundry.js:8637
    _handleCreateEmbeddedDocuments foundry.js:8637
    _createEmbeddedDocuments foundry.js:8530
    create backend.mjs:95
    createDocuments document.mjs:331
    createEmbeddedDocuments document.mjs:513
    lc ability-template.js:115
    emit index.js:202
    dispatchEvent interaction.min.js:8
    processPointerDown interaction.min.js:8
    recursiveFindHit interaction.min.js:8
    findHit interaction.min.js:8
    processInteractive interaction.min.js:8
    onPointerDown interaction.min.js:8
AutoAnimationsLayer.js:83:9
```

**Comment by otigon** [comment]
I'll have to check on my end, but does the item you are using have a circular template defined?

**Comment by benbarbour** [comment]
Sphere. It's the Fireball spell from the SRD.

**Comment by otigon** [comment]
I think I found where it's erroring out. I'll push a fix in a bit

**Comment by otigon** [comment]
Issue should be resolved with 0.3.61
